### DISCUSSION TITLE
Delete an Organization via GitHub API

### DISCUSSION THREAD

**Discussion by alishahm** [community/community#33867] [Enterprise]
Running GitHub Enterprise (version 3.6.0), I've managed to be able to create organizations via the Enterprise Admin API using the [Create An Organization](https://docs.github.com/en/enterprise-server@3.6/rest/enterprise-admin/orgs#create-an-organization) guidance. Unfortunately, I can't figure out a way to delete the organization. I've tried the DELETE call using both `api/v3/orgs/{org}` and `/api/v3/admin/organizations/{org}`. Is there an endpoint I'm missing? Or is this feature not supported using the GitHub API?

**Comment by byrneh** [comment]
There is no delete organization REST API endpoint url or GraphQL mutation that deletes an Org. If you perfrom in the GUI you will also note there are aditional [I understand] confirmation dialogues to navigate through
